core/storage/user/gae_models_test.py

```diff
@@ -347,3 +352,16 @@ def test_get_categories_where_user_can_review(self):
         self.assertIn('category1', score_categories)
         self.assertIn('category3', score_categories)
         self.assertNotIn('category2', score_categories)
+
+
+class UserSubscriptionsModelTests(test_utils.GenericTestBase):
+    """Tests for UserSubscriptionsModel."""
+    USER_ID = 'user_id'
+
+    def test_export_data(self):
+        user_data = user_models.UserSubscriptionsModel.export_data(self.USER_ID)
```

Comment:
This test does not take into account that there might be additional fields in user_data. Do an assertion on user_data itself rather than only on its fields.

Also, I suggest having a test where the values of the fields are non-trivial, to check format etc. Don't just aim for numerical test coverage; think about the behaviours you want to enforce and the things that might go wrong, and make sure that your tests cover these fully.